src/components/SearchList.js

```diff
@@ -1,11 +1,46 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import styled from 'styled-components';
 import propTypes from 'prop-types';
 
-function SearchList({ items, handleSearchQueryFromList, loading }) {
+function SearchList({
+  items,
+  activeSearchListElement,
+  onSetActiveSearchListElement,
+  setSelectedCity,
+  handleSearchQueryFromList,
+  loading,
+}) {
+  const form = document.querySelector('div #form');
```

Comment:
I think we can use `useRef` hooks here for the element's reference which  is more react way of accessing an element

Reply:
Fixed